asserts/snap_asserts.go

```diff
@@ -99,9 +99,101 @@ func buildSnapBuild(assert assertionBase) (Assertion, error) {
 	}, nil
 }
 
+// SnapRevision holds a snap-revision assertion, asserting the properties of a
+// snap release by a store authority.
```

Comment:
I suggest something along these lines:

// SnapRevision holds a snap-revision assertion, which is a statement by the store
// acknowledging the reception of a snap build and labeling it with a snap revision.